indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -47,4 +48,154 @@
             $('#payment-summary').effect('highlight', 800);
         });
     }
+
+    function setupStaticURLGeneration() {
+        $('.js-static-url').on('click', function(event) {
+            var $this = $(this);
+            $.ajax({
+                method: 'POST',
+                url: $this.data('href'),
+                success: function(result) {
+                    var content = $('.registrations > .clipboard-dialog').clone();
+                    content.find('input:text').val(result.url);
+
+                    $this.qtip({
```

Comment:
jquery.custom.js would be an option (if you add it to `$.fn` to make it callable through a jquery object)